src/Label.svelte

```diff
@@ -17,12 +17,6 @@
   export let fore;
   export { fore as for };
   export let id = '';
-  export let xs = '';
-  export let sm = '';
-  export let md = '';
-  export let lg = '';
-  export let xl = '';
-  export let widths = colWidths;
 
```

Comment:
These are actually used below on line 30:

`let columnProp = $$props[colWidth];`

Can you configure linter to ignore these?  Is this eslint?

Reply:
I think it works the same also w/o definition with `export`.

Reply:
These are official props though `<Label xs={12} sm={3}>` , shouldn't we export all props?